src/Sulu/Bundle/DocumentManagerBundle/SuluDocumentManagerBundle.php

```diff
@@ -22,10 +22,6 @@ public function build(ContainerBuilder $container)
     {
         parent::build($container);
         $container->addCompilerPass(new InitializerPass());
-        $container->addCompilerPass(new RegisterListenersPass(
-            'sulu_document_manager.event_dispatcher',
-            'sulu_document_manager.event_listener',
-            'sulu_document_manager.event_subscriber'
-        ));
+        $container->addCompilerPass(new SubscriberPass());
```

Comment:
The new subscriber pass replaces the generic `RegisterlistenersPass`, which no longer meets our requirements.